.travis.yml

```diff
@@ -22,7 +22,6 @@ branches:
 # command to install requirements
 install:
   - pip install $DJANGO
-  - pip install https://github.com/openwisp/openwisp-users/tarball/master
   - pip install https://github.com/netjson/django-netjsongraph/tarball/master
```

Comment:
can this be removed as well?

Reply:
Yes. Was just testing as I made a mistake earlier